jamfeed.com.jfsites.json

```diff
@@ -0,0 +1,24 @@
+{  
+   "providerId":"jamfeed.com",
+   "providerName":"Jamfeed",
+   "serviceId":"jf-site",
+   "serviceName":"Jamfeed Site",
+   "version": 1,
+   "logoUrl":"https://www.jamfeed.com/images/JamFeed-Logo--Text-on-Side--MEDIUM---black--green-799x163-1-p-500.png",
+   "description":"Enables a domain to be verified and work with user created Jamfeed websites",
+   "variableDescription":"verify is the verification code for the site. cert is certificate route. id is the id for the cdn distribution of the site.",
+   "records":[  
+      {  
+         "type":"CNAME",
+         "host":"%verify%",
```

Comment:
A template with a CNAME and both host and pointsTo variable is considered as potentially vulnerable to phishing.
Please refer to the spec for details.
In this context please either use the signing (preferred) or mark the template as insecure.

Reply:
Thank you Pawel for the input, i'll check on it 👍🏼 🙌🏼